azimuth/modules/base_classes/indexable_module.py

```diff
@@ -202,6 +223,40 @@ def _save_result(self, res, dm):
                 ],
                 table_key=table_key,
             )
+            dm.add_column_to_prediction_table(
+                key=DatasetColumn.pipeline_steps,
+                features=[
+                    {
+                        "preprocessing_steps": [
+                            PreprocessingStepAPIResponse(
+                                class_name=step.class_name, text=step.text[0]
+                            ).dict()
+                            for step in pred_res.preprocessing_steps
+                        ],
+                        "postprocessing_steps": [
+                            PostprocessingStepAPIResponse(
+                                class_name=step.class_name,
+                                output=PredictionDetails(
+                                    predictions=[
+                                        class_names[cl_idx]
+                                        for cl_idx in reversed(np.argsort(step.output.probs[0]))
+                                    ],
+                                    prediction=class_names[step.output.preds[0]],
+                                    confidences=[
+                                        prob for prob in reversed(np.sort(step.output.probs[0]))
+                                    ],
```

Comment:
I think it's more efficient to reverse numpy arrays using `[::-1]` than using `reverse()`. That would apply for the other `reverse()` above too. However, the gain would probably be insignificant, so feel free to keep `reverse()` if you prefer it for readability.
Also, instead of `[prob for prob in ...]` we can use `list(...)`.
```suggestion
                                    confidences=list(np.sort(step.output.probs[0])[::-1]),
```

Reply:
> `[prob for prob in ...]` we can use `list(...)`

Oh, awesome, I never realized I could simplify this. Thanks!!